app/models/ticket_purchase.rb

```diff
@@ -2,6 +2,7 @@ class TicketPurchase < ActiveRecord::Base
   belongs_to :ticket
   belongs_to :user
   belongs_to :conference
+  belongs_to :payment
```

Comment:
Then add this is in a different commit, as it to solve something that was already wrong and not really related with this feature please. And after that I would say we are ready to merge it. @mdeniz do you want to take a last look? :wink: